src/test/java/com/arpnetworking/metrics/impl/TsdQueryLogSinkTest.java

```diff
@@ -568,15 +572,21 @@ private String fileToString(final File file) throws InterruptedException {
             + "  \"version\":\"0\""
             + "}";
 
+    private static final String SCHEMA_FILE_NAME = "query-log-schema-2e.json";
+
     static {
         JsonNode jsonNode = null;
         try {
-            // Normally this is being executed from the project directory (e.g. root/client-java)
-            jsonNode = JsonLoader.fromPath("../doc/query-log-schema-2e.json");
+            // Attempt to load the cached copy
+            jsonNode = JsonLoader.fromPath("./target/" + SCHEMA_FILE_NAME);
         } catch (final IOException e1) {
             try {
-                // Under some IDE setups this may be executed from the workspace root (e.g. root)
-                jsonNode = JsonLoader.fromPath("doc/query-log-schema-2e.json");
+                // Download from the source repository
+                jsonNode = JsonLoader.fromURL(
+                        new URL("https://raw.githubusercontent.com/ArpNetworking/metrics-client-doc/master/schema/" + SCHEMA_FILE_NAME));
+
+                // Cache the schema file
+                Files.write(Paths.get("./target/" + SCHEMA_FILE_NAME), JacksonUtils.prettyPrint(jsonNode).getBytes(Charsets.UTF_8));
```

Comment:
Whoa. Does this cached copy persist a little when using travis or something? Or is this just for local builds?